libs/pipelines/api_v2_pipeline.py

```diff
@@ -267,6 +267,13 @@ def _model_to_dict(data: dict):
         if isinstance(value, pydantic.BaseModel):
             value = _model_to_dict(value.__dict__)
 
+            # When converting API model objects to dicts, we always add an
+            # "unused" entry to the dict. This allows us to use "unused" as a
+            # column in our CSV output to have a blank column. This is handy
+            # since when we remove an API column we replace it with "unused" to
+            # preserve column ordering).
+            results["unused"] = None
```

Comment:
See csv_column_ordering.py below for how this comes into play...

Reply:
I didn't see in the apidocs an update that users should expect blank columns in order to preserve column numbering even after deprecated functions.

I could imagine someone new to the dataset wondering why that is. We should add a note.

Reply:
If there's somewhere you think makes sense to add it (ideally without adding it in a bunch of places, let me know and I'll do a follow-up PR).